src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.html

```diff
@@ -0,0 +1,49 @@
+<!--
+  TODO
+  - Look for a better icon here https://fontawesome.com/v5/search
+  - Set right title, subtitle, explanation
+-->
+<app-dashboard-widget
+  icon="birthday-cake"
+  theme="child"
+  title="Birthday"
+  subtitle="Upcoming Birthdays"
+  explanation="Temporary explanation"
+>
+  <app-widget-content>
+    <div class="mat-elevation-z8">
+      <table mat-table [dataSource]="dataSource" matSort>
+        <ng-container matColumnDef="entityId">
+          <th mat-header-cell *matHeaderCellDef mat-sort-header>Child Name</th>
+          <td mat-cell *matCellDef="let children">{{ children.entityId}} </td>
+        </ng-container>
+
+        <ng-container matColumnDef="Birthdays">
+          <th mat-header-cell *matHeaderCellDef mat-sort-header>Birthday Date</th>
+          <td mat-cell *matCellDef="let children">{{children.Birthday}} </td>
```

Comment:
```suggestion
          <td mat-cell *matCellDef="let children">{{children.dateOfBirth}} </td>
```
`Birthday` does not exist on the `Child` class. Check [here](https://aam-digital.github.io/ndb-core/documentation/classes/Child.html) for more information.